exercises/concept/calculator-conundrum/.docs/instructions.md

```diff
@@ -0,0 +1,36 @@
+# Instructions
```

Comment:
In general, this needs to follow https://exercism.org/docs/building/tracks/concept-exercises

There are several files that are missing:

.docs/introduction.md: introduce the concept(s) that the exercise teaches to the student (required)
.docs/hints.md: provide hints to a student to help them get themselves unstuck in an exercise (required)
.meta/design.md: describe the design of the exercise (required)

You also need an entry in the top-level config.json at the root of the project.